File: Tema 3 -Sentencia condicional (if-switch)/ejercicios/ejercicio2 hora por teclado y salude/archives/2.py
import sys

# Obtener los argumentos de la línea de comandos
num1 = int(sys.argv[1])
if (num1 >= 0 and num1 < 24):
    if (num1 >= 6 and num1 <= 12):
        solucion = "Buenos Dias"
    elif (num1 >= 13 and num1 <= 20):
        solucion = "Buenas Tardes"
    elif (num1 >= 21 or num1 <= 5):
        solucion = "Buenas Noches"
else:
    solucion = "Hora no valida"
print(solucion)

File: Tema 3 -Sentencia condicional (if-switch)/ejercicios/ejercicio2 hora por teclado y salude/archives/2.js
function sumarJS() {
  const num1 =parseInt(document.getElementById("num1").value);
  let solucion;
  if (num1 >= 0 && num1 < 24) {
    if (num1 >= 6 && num1 <= 12) {
      solucion = "Buenos Días";
    } else if (num1 >= 13 && num1 <= 20) {
      solucion = "Buenas Tardes";
    } else if (num1 >= 21 || num1 <= 5) {
      solucion = "Buenas Noches";
    }
  } else {
    solucion = "Hora no valida";
  }

  const resultadoDiv = document.getElementById("resultadoJavascript");
  resultadoDiv.innerHTML = `${solucion}`;
}
